script/Arcueid/Day 10/10_00_ARC10LIVING.txt

```diff
@@ -818,27 +863,28 @@ Through an arch of gloomy trees, an old mansion appears.
 [sha:6d4d2f2445425b5fc0444e1c6fb51d7c0f54feaa]{
 -- Page 938, Offset 26992.
 -- 感嘆とも、<諦観|ていかん>とも取れるため息が漏れた。
-A sigh leaks out of me that could be taken as a sign of admiration or resignation.
+// Wowie, a D1 callback
+I let out a sigh that could be taken either as admiration or resignation.
 }
 [sha:10e8eae518259265e5ee4d3b94723af4026143df]{
 -- Page 939, Offset 26993.
 -- この屋敷は七年前から何も変わってはいない。
-This mansion hasn't changed at all in the past seven years.
+This mansion hasn't changed at all over the past seven years.
 }
 [sha:64749f8a537ba3d46d9a16ded187d9c49134fc4b]{
 -- Page 939, Offset 26994.
 -- 堅固な城には必ず牢が作られている。
-Within that sturdy castle, they've gotta be building a prison.
+A sturdy castle always houses a prison.
```

Comment:
Is this reading right?

Reply:
Is this not passive form ongoing? As in, a prison has been constructed within it?

Reply:
i'm not sure what my problem was with this line actually, it's fine.